lib/usd/translators/shading/usdLambertWriter.cpp

```diff
@@ -110,12 +111,21 @@ void PxrUsdTranslators_LambertWriter::Write(const UsdTimeCode& usdTime)
         }
     }
 
-    AuthorShaderInputFromShadingNodeAttr(
-        depNodeFn,
-        _tokens->incandescence,
-        shaderSchema,
-        PxrMayaUsdPreviewSurfaceTokens->EmissiveColorAttrName,
-        usdTime);
+    // Since incandescence in Maya and emissiveColor in UsdPreviewSurface are
+    // both black by default, only author it in USD if it is authored in Maya.
+    const MPlug incandescencePlug =
+        depNodeFn.findPlug(
+            depNodeFn.attribute(_tokens->incandescence.GetText()),
+            /* wantNetworkedPlug = */ true,
+            &status);
+    if (status == MS::kSuccess && UsdMayaUtil::IsAuthored(incandescencePlug)) {
```

Comment:
Maybe that check should be in PxrUsdTranslators_MaterialWriter::AuthorShaderInputFromScaledShadingNodeAttr?

Reply:
Yeah, I had had that thought too, so I'll make that change. Thanks!

Reply:
I turned this into an optional parameter to `AuthorShaderInputFromShadingNodeAttr()` in https://github.com/Autodesk/maya-usd/pull/848/commits/7db9ee89e9149f50c28ee44bf92caca9fc8cbd1d and make use of it for the `lambert` writer in https://github.com/Autodesk/maya-usd/pull/848/commits/e81ca1f3bec715894e310e89061b96607f26e9e7.